packages/docs/src/page-configs/ui-elements/button/api-options.ts

```diff
@@ -4,9 +4,6 @@ export default defineManualApi({
   slots: {
     default: { },
   },
-  events: {
-    click: { types: '`() => Event`' },
```

Comment:
`VaButton` doesn't emit click, I think it is a common need, is this a bug of `VaButton`, or am I making mistakes with this sort of events?

Reply:
VaButton doesn't emit "click" as custom component event, its true. 
When user type:
```vue
<VaButton @click="doSmth" />
```
he actually listens to DOM event, which propagate from `button` tag.

So hypothetical user can make this code which work okay.
```vue
<VaButton @dblclick="doSmth" @contextmenu="doSmth2" />
```

So there is no bug, you can click on any vuestic component.

So, this is a lie. We do not emit any clicks 😁
![image](https://user-images.githubusercontent.com/23530004/168388767-0b50335b-f08b-47a6-a93e-4251a4c075a7.png)

I think we can safety remove it from docs. 
Or we can add DOM events section under spoiler, so no one will be confused with missing events. And add all DOM Mouse events: https://developer.mozilla.org/en-US/docs/Web/Events
What do you think?

Reply:
/cc @asvae

Reply:
Thanks, @m0ksem, I'll wait for your discussion.

Reply:
Yes, it will be very confusing for the users to remove this event from docs.

Reply:
Hey @xiongmao86 

We will handle it in the other task to not bother you :)

Would be amazing if you could just add to such events description in the docs something like this in the end.

For example for this click `Event` description will be:

```
Emitted when user clicks on button
(native DOM event)
```

Thanks again 👍

Reply:
Thanks, @Derranion. I'll do what you said.